h1st/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+   "project":"Human-First AI (H1st)",
+   "author":"H1st AI",
+   "description":"Combine Human Knowledge with Machine Learning",
+   "release":"0.8.0",
```

Comment:
Pls update `release` and `version` to `0.1.1`. Also update the version in `pyproject.toml` to `0.1.1` too.